hooks/api_server.py

```diff
@@ -71,54 +71,12 @@ def on_post(self, req, resp):
             resp.status = falcon.HTTP_500
 
 
-class CorsProxy:
-    _DEFAULT_USER_AGENT = "ROI Hunter/CORS proxy; http://roihunter.com/"
-    _TIMEOUT_IN_SECONDS = (7, 10)
-
-    def __init__(self):
-        self._logger = services.logger()
-
-    def on_get(self, req, resp):
-        resp.set_header("access-control-allow-origin", "*")
-        resp.set_header("access-control-allow-methods", "GET")
-        resp.set_header("access-control-max-age", "21600")
-        allow_headers = req.get_header("access-control-request-headers")
-        if allow_headers:
-            resp.set_header("access-control-allow-headers", allow_headers)
-
-        url = req.get_param("url")
-
-        try:
-            self._logger.info("Trying to proxy URL: %s", url)
-            response = requests.get(url, timeout=self._TIMEOUT_IN_SECONDS, headers={"User-Agent": self._get_user_agent(req)})
```

Comment:
@Joozty I think you removed the dependency on `requests`. :-) Remember also the eventlet patch.

Reply:
https://github.com/business-factory/captain-hook/pull/39/commits/e431e335dd199202560b92b6354dcb282fa1f9c2

Reply:
https://github.com/business-factory/captain-hook/pull/39/commits/35ea0931bd0ff9dc694d01ac37677bb809bde56b